middlewares/logger_test.go

```diff
@@ -20,7 +20,7 @@ type logtestResponseWriter struct{}
 
 var (
 	logger                  *Logger
-	logfileName             = "traefikTestLogger.log"
+	logfileName             = "/traefik/logger/test.log"
```

Comment:
Could you rename this to `logfileNameSuffix` since it will be concatenated with the actual temp directory.